peercrawler.py

```diff
@@ -479,18 +485,6 @@ def main():
     if args.beta: ctx = betactx
     if args.test: ctx = testctx
 
-    # setup logging
-    file_name = "peercrawler.log"
-
-    logging.basicConfig(
-        level=logging.DEBUG,
-        format="%(levelname)s %(asctime)s: %(message)s",
-        handlers=[
-            RotatingFileHandler(file_name, mode="a", maxBytes=(50 * 5000), backupCount=1),
-            logging.StreamHandler()
-        ]
-    )
-
```

Comment:
The logger should be setup outside of main(), otherwise it wouldn't be configured when running the peercrawler with the web server.